### ISSUE TITLE
Questions about tICA

### ISSUE THREAD

**Issue by Juanrong** [msultan/tica_metadynamics_paper_1#4]
As I know, the first step of tICA-Metadynamics is to run some MD simulations starting from all available crystal structures and then analyze these trajectories by tICA. Muneeb told me I don't have to concatenate the trajectories and MSMBuilder will work with multiple simulations to find the slowest coordinate. A question confuses me for a very long time: Can I use tICA to analyze multiple short trajectories as PCA? For example, 100 short trajectories, 1ns for each one. Can I treat them as a 100ns trajectory and analyze it using tICA or use MSMBuilder to deal with these multiple simulations?  As I know, the connection order will influence the result of tICA and get different results if I concatenate them as a single trajectory.

**Comment by msultan** [comment]
>A question confuses me for a very long time: Can I use tICA to analyze multiple short trajectories as PCA? For example, 100 short trajectories, 1ns for each one.

yes, that should be fine. If any one of those 100 trajectories ends up sampling your process of interest, tICA will be able to pull them out. 

> Can I treat them as a 100ns trajectory and analyze it using tICA or use MSMBuilder to deal with these multiple simulations? As I know, the connection order will influence the result of tICA and get different results if I concatenate them as a single trajectory.

DO NOT concatenate the trajectories, that will completely mess up the statistics and is also an invalid trajectory on its own.

**Comment by Juanrong** [comment]
Thanks for your answers. How to analyze multiple simulations using MSMBuilder? Which function or parameter could do it? I only used MSMBuilder to analyze one trajectory one time.

**Comment by Juanrong** [comment]
I got it after reading the example of Fs peptide in MSMBuilder. The input set of tICA function could be a 3-D matrix. Is it right? If I analyze the 100 trajectories, 1ns for each one,  how much lag time should I choose? 0.1ns?

**Comment by msultan** [comment]
>I got it after reading the example of Fs peptide in MSMBuilder. The input set of tICA function could be a 3-D matrix. Is it right? If I analyze the 100 trajectories, 1ns for each one, how much lag time should I choose? 0.1ns?

no, its a **list of 2-D matrices**. Each matrix in that list is time points by protein features (dihedrals,contacts, angles etc).

**Comment by Juanrong** [comment]
Is it necessary that the simulation time of all available structures for tICA in the first step of tICA-Metadynamics must be same? I only know the theory of tICA dealing with one time series and how does it handle multiple time-series at one time?

**Comment by msultan** [comment]
>Is it necessary that the simulation time of all available structures for tICA in the first step of tICA-Metadynamics must be same?

not really, I think it was easier for me to set it up that way but you can start the tica-metadynamics situations from any set of points across your gpus. 

>I only know the theory of tICA dealing with one time series and how does it handle multiple time-series at one time? 

its fit exactly the same way. take a look at the Src kinase example in msmbuilder where we fit a model using 20000 trajectories that were all of different lengths and started from different seed pdbs.

**Comment by DanielWicz** [comment]
> 
> 
> > Is it necessary that the simulation time of all available structures for tICA in the first step of tICA-Metadynamics must be same?
> 
> not really, I think it was easier for me to set it up that way but you can start the tica-metadynamics situations from any set of points across your gpus.
> 
> > I only know the theory of tICA dealing with one time series and how does it handle multiple time-series at one time?
> 
> its fit exactly the same way. take a look at the Src kinase example in msmbuilder where we fit a model using 20000 trajectories that were all of different lengths and started from different seed pdbs.

Do you mean this paper ? https://www.sciencedirect.com/science/article/pii/S0006349516310487

I don't where it refers to the theory, how TICA are constructed for non-continuous trajectory. Could refert to the part, where it's explained ?
